doc/contribute/onboarding.rst

```diff
@@ -0,0 +1,102 @@
+Onboarding
+==========
+
+This page aims to be a list of resources that should be helpful if you want to start contributing to CCF.
+
+Introduction
+------------
+
+Start by reading :doc:`/overview/what_is_ccf`.
+
+If you encounter any terms or acronyms that you don't know, check the :doc:`/overview/glossary`. If the word you are looking for isn't defined, create an `issue <https://github.com/microsoft/CCF/issues/new/choose>`_.
+
+Create an SGX VM if necessary
+-----------------------------
+
+If you intend to make changes that need to work on Intel SGX, you will want an :doc:`SGX VM </contribute/create_vm>` to check you didn't introduce instructions that are illegal in enclave, and to evaluate the performance impact of your change.
+
+Build CCF
+---------
+
+First complete :doc:`/contribute/build_setup`, and see :doc:`/contribute/build_ccf`.
+
+Walk through a sample application
+---------------------------------
+
+:doc:`/build_apps/logging_cpp` documents the sample native app under ``/samples/apps/logging`` in the repo.
+
+The logging application is simple, but exercises most features of the framework, and is extensively used in the end to end tests.
+
+To run a locally built copy of this application in a sandbox, see :doc:`/build_apps/run_app`. The package name will be ``samples/apps/logging/liblogging``:
+
+.. code-block:: bash
+
+    ~/CCF/build$ ../tests/sandbox/sandbox.sh -p samples/apps/logging/liblogging
```

Comment:
Maybe link to here to avoid duplication https://microsoft.github.io/CCF/main/build_apps/run_app.html?highlight=sandbox?

Reply:
There is already a link on line 30, but I deliberately added this invocation because it's not quite the same in two important ways:

1. it's done for a local build, not install of CCF
2. it points at the non-obviously located liblogging package, which is now three levels deep in a subdirectory because of CMake

Reply:
I'd missed that, thanks.